src/main/java/org/hyperledger/fabric/sdk/security/CryptoPrimitives.java

```diff
@@ -207,61 +206,51 @@ public Certificate bytesToCertificate(byte[] certBytes) throws CryptoException {
      * @param pemCertificate
      * @return
      */
-
-    private X509Certificate getX509Certificate(byte[] pemCertificate) throws CryptoException {
+    private X509Certificate getX509CertificateInternal(byte[] pemCertificate) throws CryptoException {
         X509Certificate ret = null;
         CryptoException rete = null;
 
         List<Provider> providerList = new LinkedList<>(Arrays.asList(Security.getProviders()));
-        if (SECURITY_PROVIDER != null) { //Add if overridden
```

Comment:
Clarified that adding a security provider may not do much. If you add it you probably want it preferred which means adding it to the start of the list, not the end.